releases/v1.1.0-rc.0.toml

```diff
@@ -0,0 +1,51 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.0"
+
+pre_release = true
+
+preface = """\
+`containerd` provides a daemon for managing running containers.
+
+1.1 is the second major release for `containerd` with added support for CRI, the
+Kubernetes Container Runtime Interface. CRI is a new default plugin which allows
```

Comment:
Do we want to add a link to "Container Runtime Interface"?
https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md

Reply:
We do, do you also have a link for configuring the Kubelet to use a non-default CRI runtime?